### PR TITLE
weld3 integration code

### PR DESCRIPTION
**Pull request by ankathur**

### CODE REVIEW

File: main/appserver/pom.xml [hunk 1]
```diff
@@ -0,0 +1,1119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (arindam-bandyopadhyay) [inline main/appserver/pom.xml]: There is no main/ anymore.

**Comment by ankathur** [comment]
@glassfishrobot Run CI tests please

File: main/appserver/web/weld-integration/src/test/java/org/glassfish/weld/services/TransactionServicesImplTest.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
```
Comment (arindam-bandyopadhyay) [inline main/appserver/web/weld-integration/src/test/java/org/glassfish/weld/services/TransactionServicesImplTest.java]: There is no main/

**Review by arindam-bandyopadhyay — CHANGES_REQUESTED** [review]
Please remove main/ from the PR

**Comment by ankathur** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — CHANGES_REQUESTED** [review]
Please fix the license URL in EjbInterceptorTest.java and TransactionServicesImplTest.java

**Comment by ankathur** [comment]
@glassfishrobot Run CI tests please

**Comment by ankathur** [comment]
@glassfishrobot Run CI tests please

File: appserver/deployment/dol/src/test/java/com/sun/enterprise/deployment/EjbInterceptorTest.java [hunk 3]
```diff
@@ -8,7 +8,7 @@
  * and Distribution License("CDDL") (collectively, the "License").  You
  * may not use this file except in compliance with the License.  You can
  * obtain a copy of the License at
- * https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
+ * https://oss.oracle.com/licenses/CDDL+GPL-1.1
  * or packager/legal/LICENSE.txt.  See the License for the specific
```
Comment (yaminikb) [inline appserver/deployment/dol/src/test/java/com/sun/enterprise/deployment/EjbInterceptorTest.java]: License text path needs to be fixed.

**Review by yaminikb — CHANGES_REQUESTED** [review]

File: appserver/deployment/dol/src/test/java/com/sun/enterprise/deployment/EjbInterceptorTest.java (hunk 3 above)
Comment (ankathur) [inline appserver/deployment/dol/src/test/java/com/sun/enterprise/deployment/EjbInterceptorTest.java]: Fixed.

**Pull request merged**